src/bin/main.rs

```diff
@@ -14,16 +14,16 @@ extern crate env_logger;
 extern crate getopts;
 extern crate rustfmt_nightly as rustfmt;
 
-use std::{env, error};
```

Comment:
I am not sure if this and similar changes are correct. I feel that `use std::{env, error};` should have higher precedence than `use std::fs::File;` and other imports that start with `use std::`.

Reply:
Hmm, I was pretty much making it up as I went along, but this was intentional. Looking at nested imports, I prefer `use foo::{bar::baz, baz::{a, b}};` to the opposite ordering. I can't really justify why, but it felt to me like single imports should come before list imports. Do you have a reason for supporting the other order?

Reply:
Hmm, actually inside nested imports I prefer to put list imports after single imports, but on top level I prefer the opposite.

Maybe it is just that I am used to the current behavior. To keep the rule consistent I agree that we should put the list imports after single imports.